apps/proxy/src/app/layout/layout.component.ts

```diff
@@ -17,6 +17,8 @@ import { CookieService } from 'ngx-cookie-service';
 export class LayoutComponent extends BaseComponent implements OnInit {
     public toggleMenuSideBar: boolean;
     public logInData$: Observable<IFirebaseUserModel>;
+    public sideNavOpenState = true;
+    public isSideNavOpen$ = new BehaviorSubject<boolean>(this.sideNavOpenState);
```

Comment:
```suggestion
    public isSideNavOpen$ = new BehaviorSubject<boolean>(true);
```

Reply:
Done

Reply:
Only observable name should end with `$`, for a subject it should just be `isSideNavOpen`